be/src/runtime/jdbc_driver_manager.h

```diff
@@ -0,0 +1,58 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include <memory>
+#include <mutex>
+#include <unordered_map>
+
+#include "common/status.h"
+
+namespace starrocks {
+
+struct JDBCDriverEntry;
+
+// JDBCDriverManager is responsible for managing jdbc driver jar files.
+// All jar files will be placed in `${STARROCKS_HOME}/lib/jdbc_drivers` and named in the format of ${name}_${checksum}_${first_access_ts}.jar
+// `first_access_ts` represents the time when the driver is accessed for the first time on this node.
```

Comment:
The jdbc jar files only need download once in most of cases?  why not could we unify the download method with UDF？

Reply:
> The jdbc jar files only need download once in most of cases? why not could we unify the download method with UDF？

indeed. So I unified the download logic into the interface `DownloadUtil::download`